test/test-mysql-index.js

```diff
@@ -6,8 +6,9 @@ const assert = require('assert');
 const sinon = require('sinon');
 const mysql = require('mysql');
 
-const index = require('../lib/mysql/index');
 const cst = require('../lib/mysql/constants');
+const index = require('../lib/mysql/index');
+const queries = require('../lib/mysql/queries.json');
```

Comment:
As creating an object from a JSON file is different from important a library, I would add a new line to separate the two.